samples/drivers/adc/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_LOG=y
+CONFIG_ADC_LOG_LEVEL_DBG=y
+CONFIG_DMA_LOG_LEVEL_DBG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
```

Comment:
Only used during debugging.
```suggestion
```

Reply:
removed

Reply:
changed to CONFIG_NO_OPTIMIZATIONS

Reply:
We can't rely on `CONFIG_NO_OPTIMIZATIONS=y`. In some of our optima apps the application doesn't fit in FLASH without optimizations.